examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -142,6 +151,8 @@ export default class App extends PureComponent {
     this.handleClosePanel = this.handleClosePanel.bind(this);
     this.handleSelectTileColor = this.handleSelectTileColor.bind(this);
     this.handleClearWarnings = this.handleClearWarnings.bind(this);
+    this.handleEnableNormalsVisualisation = this.handleEnableNormalsVisualisation.bind(this);
```

Comment:
That is a mouthful... `handleShowNormals`?

Reply:
Renamed.